modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
Here's a suggested flow which makes sense to me:
1) generate no implementation dependency by default, this will generate a "clean" version, add additional info / sample code on how to inject own HTTP client
2) generate an additional class if `--library=httplug` is provided, this version is basically 1) with an additional item in composer.json and a HTTP client pre-injected for you (so, you can still inject another client yourself if you don't want to use the additional class, it just doesn't install a depenency you might not want by default)

We might make 2) the default to keep BC though.